infra/base-images/base-builder/Dockerfile

```diff
@@ -27,7 +27,7 @@ ENV SANITIZER_FLAGS_memory "-fsanitize=memory -fsanitize-memory-track-origins"
 ENV SANITIZER_FLAGS_profile "-fprofile-instr-generate -fcoverage-mapping -pthread"
 
 # Default build flags for coverage.
-ENV COVERAGE_FLAGS="-fsanitize-coverage=trace-pc-guard,trace-cmp"
+ENV COVERAGE_FLAGS="-fsanitize=fuzzer-no-link"
 
 # Coverage flags for generating coverage reports.
```

Comment:
I guess we don't want to change that, do we?

Reply:
At least not now.